integration/src/test/scala/org/knora/webapi/responders/admin/UsersResponderSpec.scala

```diff
@@ -65,74 +76,71 @@ class UsersResponderSpec extends CoreSpec with ImplicitSender {
         assertFailsWithA[ForbiddenException](exit)
       }
     }
-  }
 
-  "The UsersResponder " when {
-    "asked about an user identified by 'iri' " should {
+    "calling getUserByEmail" should {
+
+      def getUserByEmail(email: Email, requestingUser: User): ZIO[UsersRestService, Throwable, UserResponseADM] =
+        ZIO.serviceWithZIO[UsersRestService](_.getUserByEmail(requestingUser, email))
+
       "return a profile if the user (root user) is known" in {
         val actual = UnsafeZioRun.runOrThrow(
-          UsersResponder.findUserByIri(
-            UserIri.unsafeFrom(rootUser.id),
-            UserInformationTypeADM.Full,
-            KnoraSystemInstances.Users.SystemUser
-          )
+          getUserByEmail(Email.unsafeFrom(rootUser.email), KnoraSystemInstances.Users.SystemUser)
         )
-        actual shouldBe Some(rootUser.ofType(UserInformationTypeADM.Full))
+        actual.user shouldBe rootUser.ofType(UserInformationType.Restricted)
```

Comment:
**question:** why does that change from full to restricted?

Reply:
Internally the responder always returned the fully filled  `User` entity, like the `UserService` would do now. 

However, on API we always remove the password hash using the `Restriced` info type for security purposes. The test is now using the `RestService` so that is why we have to assert the API representation now.

Reply:
ok, that makes sense. Thanks for the explanation!